### ISSUE TITLE
CST - Status details page error state has no H1

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#108003]
## Issue
There is no H1 on the Status details page when there is an error/nothing loads. "We're sorry, something went wrong on our end" could be the H1.

New Cypress Axe rules for headings are coming and will be put into effect April 30. [Slack thread](https://dsva.slack.com/archives/C03R5SBELQM/p1742920275472879) The link to the Axe report expired, but I noticed this one while running an Axe Devtools scan.

Page: ex: https://staging.va.gov/track-claims/appeals/5157699/status

<img width="1380" alt="Image" src="https://github.com/user-attachments/assets/29bed159-094a-4d61-bd0b-dbc7533f2c32" />
